### ISSUE TITLE
fault in using cross encoder

### ISSUE THREAD

**Issue by ReySadeghi** [UKPLab/sentence-transformers#702]
I trained a cross encoder model and my label is (0 or 1) and in term of using "cross-encoder_usage.py ", I got this error:

Traceback (most recent call last):
  File "predict.py", line 39, in <module>
    print("{:.2f}\t{}".format(similarity_scores[idx], corpus[idx]))
TypeError: only integer scalar arrays can be converted to a scalar index

and I have a question about type of "similarity_scores", is it an array of 2?
it shouldn't be a number according to my label?

**Comment by nreimers** [comment]
Did you define num_labels=1 or num_labels=2

For num_labels=1, singular values between 0 and 1 are returned

for num_labels=2, two values are returns: The first the logits for label 0, the second the logits for label 1

**Comment by ReySadeghi** [comment]
yeah , I defined num_labels=2 because my train data labels are 1(paraphrase) and 0(non-paraphrase).
so it means The first is the probability for being label 0, the second is the probability for being label 1?

**Comment by nreimers** [comment]
Right.
Note, for 2 labels (binary tasks) you can also set num_labels=0. It is equivalent. Then you get a score between 0 and 1, with scores close to 0 meaning non-paraphrase and scores close to 1 meaning paraphrase.

**Comment by ReySadeghi** [comment]
ok. thanks
